.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.7.8
+    hooks:
+    -   id: flake8
```

Comment:
Does this follow the configuration in `examples/.flake8` when linting examples?

Reply:
Well right now this only checks for one single error (`F401`: unused imports) which is also compatible with `examples/.flake8`. Now it's more a proof of concept for flake8, I'd be happy extend it once we have this pre-commit config file merged :)